test/functional/editor_controller_test.rb

```diff
@@ -45,8 +45,10 @@ def setup
         tags: 'question:question,one',
         template: 'question',
         redirect: 'question'
-    assert_select "h3", "Ask a question of the community"
-    assert_select "input#taginput[value=?]", "question:question,one"
+        assert_response :redirect
```

Comment:
Sadly they don't seem to pass when I uncomment them either. And as far as I can tell the `h3` is there...the `<input>` I am not entirely sure on but seems to be referencing tags with questions so sort of at a loss.